src/wirecloud/platform/iwidget/utils.py

```diff
@@ -69,19 +70,19 @@ def process_initial_value(vardef: Union[MACDProperty, MACDPreference],
     return value
 
 
-async def update_widget_value(db: DBSession, iwidget: WidgetInstance, data: Union[IWidgetDataCreate, IWidgetDataUpdate], user: UserAll,
-                              required=False) -> Optional[CatalogueResource]:
+async def update_widget_value(db: DBSession, iwidget: WidgetInstance, data: Union[WidgetInstanceDataCreate, WidgetInstanceDataUpdate], user: UserAll,
+                              required: bool = False) -> Optional[CatalogueResource]:
     if data.widget != '' and data.widget is not None:
         (widget_vendor, widget_name, widget_version) = data.widget.split('/')
         resource = await get_catalogue_resource(db, widget_vendor, widget_name, widget_version)
         if resource is None:
-            raise ValueError('Widget not found')
+            raise ValueError(_('Widget not found'))
 
         if not await resource.is_available_for(db, user):
-            raise NotFound('Widget not available')
+            raise NotFound(_('Widget not available'))
 
         if resource.resource_type() != 'widget':
-            raise ValueError('%(uri)s is not a widget' % {"uri": data.widget})  # TODO translate this
+            raise ValueError(_('%(uri)s is not a widget' % {"uri": data.widget}))
```

Comment:
La traducción debe aplicar solo a la string sin reemplazar, es decir, sería:
```suggestion
            raise ValueError(_('%(uri)s is not a widget') % {"uri": data.widget})
```